functions-and-scope.js

```diff
@@ -72,6 +128,20 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 9
 
+// -  loop alles af met een for loop, check of de hoogstopgeslagen waarde hoger of lager is dan de huidige waarde - sla hoogste waarde op
+
+
+
+    let teller3 = null;
```

Comment:
Wanneer je met een for-loop gaat bepalen wat de hoogste (of laagste) waarde is, dan is het beter om hier de eerste waarde uit de array te pakken
```suggestion
    let teller3 = grades[0];
```

Met rapportcijfers maakt het niet zo veel uit, maar stel je voor dat je deze functie had aangeroepen met de array `[-1, -3, -8, -4]`. Dan had de uitkomst van deze functie `null` geweest.